specreduce/utils/synth_data.py

```diff
@@ -1,11 +1,13 @@
 # Licensed under a 3-clause BSD style license - see ../../licenses/LICENSE.rst
 
+from typing import Sequence
+
 import numpy as np
 
 from photutils.datasets import apply_poisson_noise
 
 import astropy.units as u
-from astropy.modeling import models
+from astropy.modeling import Model, models
```

Comment:
sorry I'm a bit confused about this - you're importing 'Model' but also redefining 'Model' below.

Reply:
i am importing `Model` to use as a type declaration and where it looks like i'm redefining it, i am actually defining the default `Model` to use as input.